src/remove-dulplicates/index.js

```diff
@@ -1,3 +1,34 @@
-function removeDuplicates(obj) {}
+function removeDuplicates(obj) {
+  // Create an array from the object values and create a copy also.
+  let arrFromObj = Object.values(obj);
+  let arrFromObjCopy = arrFromObj.map((x) => x);
```

Comment:
You can make a `shallow` copy with `[...arrFromObj]` or `arrFromObj.slice()` rather than a map.